### ISSUE TITLE
Scrollbars don’t appear in the sidebar in Chrome on macOS

### ISSUE THREAD

**Issue by adamschwartz** [cloudflare/workers-docs-engine#137]
On macOS in Chrome, if you have the system setting to show scrollbars "Automatically based on mouse or trackpad", then scrollbars don’t appear properly in the sidebar.

This is due to a Chromium bug which appeared last year involving a scrolling box containing a positioned child and a child with clip-path. This is tricky to work around since clip-path is necessary for visually hidden screen-reader text.

Chromium issue:
https://bugs.chromium.org/p/chromium/issues/detail?id=997607

Here’s a similar bug reported in a WordPress project:
https://github.com/WordPress/gutenberg/issues/16966

**Comment by adamschwartz** [comment]
In my testing, this bug exists in Chrome 84 (current stable as of Aug 7, 2020), but is fixed in Chrome 86. So there is some hope that perhaps the issue will be resolved soon.

In the meantime, for any users experiencing this issue, the current recommendation is to change your system setting

<img width=668 alt="macOS System Settings" src=https://user-images.githubusercontent.com/154613/89697994-df645f00-d8ec-11ea-8aa8-f06fc6fe8d94.png>

**Comment by adamschwartz** [comment]
Another workaround is to use a Chrome extension (or other means) to apply the following CSS to the document:

```css
.DocsSidebar [is-visually-hidden] {
  clip-path: none;
}
```

Of course theoretically we could apply this to the docs ourselves, but I’m concerned about the accessibility tradeoff without exploring this further. If anybody has experience with handling `[is-visually-hidden]` and knows of an acceptable implementation which doesn’t require `clip-path`, please let us know!

**Comment by adamschwartz** [comment]
Something like this may be an acceptable short-term trade-off:

```css
/* TODO: remove once Chromium bug is resolved:
   https://bugs.chromium.org/p/chromium/issues/detail?id=997607
*/
.DocsSidebar [is-visually-hidden] {
  clip-path: none;
  position: absolute;
  left: -10000px;
  top: auto;
  width: 1px;
  height: 1px;
  overflow: hidden;
}
```

**Comment by adamschwartz** [comment]
Good news: not long after [my comment](https://bugs.chromium.org/p/chromium/issues/detail?id=997607#c11) on [Chromium#997607](https://bugs.chromium.org/p/chromium/issues/detail?id=997607), a dev [commented](https://bugs.chromium.org/p/chromium/issues/detail?id=997607#c12) that it was resolved in M85, marked as resolved, and said it will be released as stable version soon.

**Comment by adamschwartz** [comment]
Decided to apply the temporary workaround, which we can revert as soon as the fix in [Chromium#997607](https://bugs.chromium.org/p/chromium/issues/detail?id=997607) makes it to stable Chrome (hopefully soon).
